src/metrics/listener.ts

```diff
@@ -167,7 +174,9 @@ export class MetricsListener {
   }
 
   public sendDistributionMetric(name: string, value: number, forceAsync: boolean, ...tags: string[]) {
-    this.sendDistributionMetricWithDate(name, value, new Date(Date.now()), forceAsync, ...tags);
+    // The Extension doesn't support distribution metrics with timestamps. Use sendDistributionMetricWithDate instead.
+    const metricTime = this.isExtensionRunning ? new Date(0) : new Date(Date.now());
```

Comment:
What's the purpose of setting a metric time if the extension is still gonna ignore it? (as understood by the purpose of the PR)

Reply:
I wasn't too sure if changing the type for this function's parameter would introduce a breaking change to users or not, so I figured I'd rather just not change the type of the `metricTime` param from `Date` -> `Date | undefined`. But if it's not going to change much then I can just update the type to `Date | undefined` and pass undefined here

Reply:
I think the workaround makes sense. Changing the API would definitely make a breaking change, so that's a no in the mean time. Can you leave a note in the form of `// TODO: Next breaking change, change API for metric`.

Reply:
yessir 👍